### ISSUE TITLE
Use Github for releases

### ISSUE THREAD

**Issue by Merith-TK** [NERvGear/SAO-Utils#29]
Okay so this is mainly to help get it to more people, as i live in the USA
I have the steam version, however i want to get the expiriemental version, problem is, when i go to download, it tells me "access denied"
![image](https://user-images.githubusercontent.com/10422110/92308680-4a3ca080-ef54-11ea-9c76-224ed593c2bd.png)

So, if the releases could be made availiable on this github repo, that would be fantastic. Or even have them as a "beta" option in steam!

**Comment by zapoqx** [comment]
Interesting given that I downloaded the other day no vpn as I was considering trying the newer version randomly and I live in the USA.  You sure you don't have an antivirus blocking the address or something as I have tested it just now and it works again no problem.

**Comment by Merith-TK** [comment]
@zapoqx you able to download the normal version? or the EXP version?
http://oss.gpbeta.com/sao-utils/sao_utils_exp1_update2.exe
If so, Please upload here for me, because it just gives me access denied

**Comment by Merith-TK** [comment]
Wait WTF, now it wants to download?!

**Comment by zapoqx** [comment]
ah so isp/dns provider I am guessing.

**Comment by Merith-TK** [comment]
no, i use Cloudflare for DNS, and my ISp doesnt block anything except torrent

**Comment by zapoqx** [comment]
then I have no idea what to tell ya.

**Comment by Merith-TK** [comment]
You know where i can get that alfheim online theme that is in the steam dev logs?

**Comment by zapoqx** [comment]
I'd say the workshop, but I don't see anything there related to ALO.  So maybe its on the forums?

**Comment by GPBeta** [comment]
There's also a Google Drive mirror download address inside the "Portable Version" button.

And the ALO theme mentioned in dev logs is for SAO Utils 2 which is not released yet ;P

**Comment by Merith-TK** [comment]
alright, i still say you should make releases here on github as well, as it would make it easier to get builds out to people who only understand english. 

*blaming american schooling for crappy education and teaching us things that they claim i will use in real life, when a few months after graduating, all i remember is the bullcrap my teachers gave me and none of the work*

**Comment by GPBeta** [comment]
OK, done 😀
